packages/cli-build/src/wait.js

```diff
@@ -124,12 +124,12 @@ function failureMessage(type, {
 
 // Return true or false if a build is considered failing or not
 function isFailing({
-  attributes: { state, 'total-comparisons-diff': diffs } = {}
+  attributes: { state, 'review-state': reviewState, 'total-comparisons-diff': diffs } = {}
```

Comment:
* we'd want this behind an additional flag something like `failOnChangesPassIfApproved` 
`--fail-on-changes-pass-if-approved`, this is to avoid regression for existing users.

Reply:
done! @itsjwala